libs/networking/WordPressNetworking/src/main/java/org/wordpress/android/networking/LanguageUtils.java

```diff
@@ -0,0 +1,63 @@
+package org.wordpress.android.networking;
+
+import android.content.Context;
+import android.text.TextUtils;
+
+import java.util.HashMap;
+import java.util.Locale;
+
+/**
+ * Methods for dealing with i18n messages
+ */
+public class LanguageUtils {
+
+    public static String getCurrentDeviceLanguage(Context context) {
```

Comment:
Did this and changed references properly wherever `Locale.getDefault()` was used (_and_ made sense to use the new method)